manifests/setup.pp

```diff
@@ -46,10 +46,6 @@
     default   => $script_path
   }
 
-  File {
-    backup => false,
-  }
```

Comment:
Resource defaulting has caused so many problematic side effects I'm happy to see it go...